python/private/pypi/extension.bzl

```diff
@@ -86,7 +62,35 @@ You cannot use both the additive_build_content and additive_build_content_file a
             whl_mods = whl_mods,
         )
 
-def _create_whl_repos(module_ctx, pip_attr, whl_map, whl_overrides, group_map, simpleapi_cache, exposed_packages):
+def _create_whl_repos(
+        module_ctx,
+        *,
+        pip_attr,
+        whl_map,
+        whl_overrides,
+        group_map,
+        simpleapi_cache,
+        exposed_packages,
+        whl_libraries,
+        available_interpreters = INTERPRETER_LABELS,
+        simpleapi_download = simpleapi_download):
+    """create all of the whl repositories
+
+    Args:
+        module_ctx: TODO
```

Comment:
Please add docs

IMHO just a simple type description would suffice

Reply:
Did that and instead of modifying values in places I decided to return the values and sort them properly because the lock file would get constantly changed.